<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouter_machine</class>
 <widget class="QDialog" name="ajouter_machine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/sig/erp.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="iconnes.qrc">:/sig/images/erp.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>210</y>
     <width>131</width>
     <height>241</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>27</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>19</number>
   </attribute>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>1351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="id_machine_LE">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>date de fabrication</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="type_LE">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="remarques_LE">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>marque</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_fabrication_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>441</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2014</year>
     <month>1</month>
     <day>2</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouter_mach_PB">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>590</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>ajouter_mach</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>495</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>remaques</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>id_machine</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="marque_LE">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>380</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Gere_mach_PB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>gerer machies</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ajouter une machine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler_PB">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>590</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>annuler</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>chef service maintenance</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iconnes.qrc"/>
 </resources>
 <connections/>
</ui>
